### PR TITLE
Added shear stress and axial force fucntions in continuum physics

### LINKED ISSUE #16366: No function to calculate shear stress and axial stress
In `beam.py` file of `continuum_mechanics` folder there is no function to calculate shear stress and axial stress.

### PR DESCRIPTION
**Pull request by abhinavsagar**
#### References to other Issues or PRs

[Fixes #16366 ](https://github.com/sympy/sympy/issues/16366)

#### Brief description of what is fixed or changed
Added shear stress and axial force fucntions in continuum physics .

#### Release Notes

* functions
  * added fucntion for calculating shear stress.
  * added fucntion for calculating axial stress.

### CODE REVIEW

**Review by divyanshu132 — CHANGES_REQUESTED** [review]

File: sympy/physics/continuum_mechanics/beam.py [hunk 1]
```diff
@@ -780,6 +782,74 @@ def shear_force(self):
...
+        >>> b.apply_load(R1, 10, -1)
+        >>> b.apply_load(R2, 30, -1)
+        >>> b.apply_load(120, 30, -2)
+        >>> b.bc_deflection = [(10, 0), (30, 0)]
+        >>> b.solve_for_reaction_loads(R1, R2)
+        >>> b.axial_force()
+        -8*SingularityFunction(x, 0, 0) + 6*SingularityFunction(x, 10, 0) + 120*SingularityFunction(x, 30, -1) + 2*SingularityFunction(x, 30, 0)
+        """
+        x = self.variable
+        return integrate(self.axial_force, x)
```
[the remark this answers is not in the document]
Comment (abhinavsagar) [reply sympy/physics/continuum_mechanics/beam.py]: @divyanshu132 I think there are already no trailing whitespaces. If there is can you show me where they are

[the remark this answers is not in the document]
Comment (divyanshu132) [reply sympy/physics/continuum_mechanics/beam.py]: Have a look at travis log.
`AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/physics/continuum_mechanics/beam.py, line 851.`

[the remark this answers is not in the document]
Comment (abhinavsagar) [reply sympy/physics/continuum_mechanics/beam.py]: @divyanshu132 I am still not getting it. I know travis log is showing but if it was whitespaces problem it should be at line 783 and line 817 also right?

[the remark this answers is not in the document]
Comment (divyanshu132) [reply sympy/physics/continuum_mechanics/beam.py]: Its not like that, I would suggest to use rich text editors like Sublime or something else where you can easily get rid of these whitespaces.

[the remark this answers is not in the document]
Comment (abhinavsagar) [reply sympy/physics/continuum_mechanics/beam.py]: Ok thanks for the help

**Pull request closed without merging**